go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/osmosis-labs/osmosis
 
-go 1.15
+go 1.17
```

Comment:
Why change this?

Reply:
I changed it today because of the new release, and my desire to use `embed`.

Reply:
What is `embed`? Also `go.1.17` is imo too recent, I definitely think we should be staying on 15/16 for awhile, unless quite important.